src/platform/toeplitz.c

```diff
@@ -1,170 +1,166 @@
-/*++
```

Comment:
Newline style changed?

Reply:
I have no clue. This will hopefully fix once I merge that other PR. Until then, I'm dealing with the merges...